src/table/table.scss

```diff
@@ -122,265 +249,341 @@
         }
       }
 
-      &.iui-tables-head {
-        @include themed {
-          background-color: t(iui-color-background-3);
-        }
-        user-select: none;
+      // #region Indented rows
+      &.iui-expander-indent-0 {
+        padding-left: 0;
+      }
 
-        .iui-icon {
-          @include themed {
-            fill: t(iui-icons-color-actionable);
-          }
+      &.iui-expander-indent-1 {
+        padding-left: $iui-xl;
+      }
 
-          &:hover {
-            @include themed {
-              fill: t(iui-icons-color-actionable-hover);
-            }
-          }
-        }
+      &.iui-expander-indent-2 {
```

Comment:
what is this 0, 1, 2??

Reply:
A typical cell's padding:
![Screen Shot 2021-05-18 at 4 16 23 PM](https://user-images.githubusercontent.com/849817/118717765-4e0b0d00-b7db-11eb-85b4-5497062bad2e.png)

When a cell has an expandable arrow within it, apply `.iui-expander-indent-0` to remove left padding.  Notice that the chevron arrow aligns centered on the file icon underneath & the alpha text left aligns with the icon above it:
![Screen Shot 2021-05-18 at 4 18 13 PM](https://user-images.githubusercontent.com/849817/118717921-827ec900-b7db-11eb-9c23-bb5e3f5dd8f5.png)

When a cell has an expandable arrow within it AND it is indented, apply `.iui-expander-indent-0` to increase left padding:
![Screen Shot 2021-05-18 at 4 20 20 PM](https://user-images.githubusercontent.com/849817/118718276-e73a2380-b7db-11eb-9ac6-7b11c267b501.png)

But you need to use special classes if the cell does not contain an expandable arrow, such as `.iui-indent-2`:
![Screen Shot 2021-05-18 at 4 21 49 PM](https://user-images.githubusercontent.com/849817/118718382-046ef200-b7dc-11eb-95f7-d18c15919bdb.png)


It's madness.

Reply:
I think padding in this case could be inline - the value is kind of dynamic and will most likely exceed indent level of 2. It could be calculated by iTwinUI-React Table component and nesting level should be either provided or calculated.

Or maybe something like this could be a solution: https://codepen.io/jonathanmendes/pen/aWMOrM where multiple iui-indent-* classes are generated by mixins.

Reply:
This is too complicated, IMO...
Is table a recommended approach for folder trees to display?

Reply:
To try to simplify the PR I've removed the multi-nested table rows HTML and CSS.  Better saved for another day.